.gitattributes

```diff
@@ -1,4 +1,4 @@
-deploy/charts/approver-policy/templates/crds/* linguist-generated=true
+deploy/charts/approver-policy/templates/crd-* linguist-generated=true
```

Comment:
These changes are part of https://github.com/cert-manager/approver-policy/pull/316.
This PR depends on that PR, because the generation logic for the CRDs also generates the conditional inclusion of the CRDs.